tests/test_estimation.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pymc3 as pm
+import theano.tensor as tt
+
+from pymc3_hmm.distributions import DiscreteMarkovChain, PoissonZeroProcess
+from pymc3_hmm.step_methods import FFBSStep
+
+
+def simulate_poiszero_hmm(
+    N,
+    observed,
+    mu=10.0,
+    pi_0_a=np.r_[1, 1],
+    p_0_a=np.r_[5, 1],
+    p_1_a=np.r_[1, 1],
+    pi_0=None,
+):
+    p_0_rv = pm.Dirichlet("p_0", p_0_a)
+    p_1_rv = pm.Dirichlet("p_1", p_1_a)
+    P_tt = tt.stack([p_0_rv, p_1_rv])
+    P_rv = pm.Deterministic("P_tt", tt.shape_padleft(P_tt))
+    if pi_0 is not None:
+        pi_0_tt = tt.as_tensor(pi_0)
+    else:
+        pi_0_tt = pm.Dirichlet("pi_0", pi_0_a)
+    S_rv = DiscreteMarkovChain("S_t", P_rv, pi_0_tt, shape=N)
+    S_rv.tag.test_value = (observed > 0) * 1
+    return PoissonZeroProcess("Y_t", mu, S_rv, observed=observed)
```

Comment:
As a simple test for initial probability estimation, I don't know if the zero-Poisson model is really the best choice.

If we're going to use a model with a Dirac emission distribution, we should use one that consists entirely of Dirac emissions; it's simpler and accomplishes the same thing.

Otherwise, we need to consider whether or not a non-Dirac model is a better choice for testing (e.g. mixture of normals).  Likewise for the choice of transition matrix.